### ISSUE TITLE
bad advice for using sudo

### ISSUE THREAD

**Issue by richRemer** [npm/docs#852]
Currently the "08 - Installing npm packages globally" page gives advice to avoid sudo when installing packages globally, and directs the user to several workarounds intended to open up permissions for system files or to point npm at a non-system directory.

This is not how system files are generally handled on Linux systems, and leads to perennial problems for npm users getting EACCES errors (*q.v.*, https://stackoverflow.com/questions/16151018/npm-throws-error-without-sudo), and there is absolutely no help for system administrators trying to operate a shared system.

Should instead advise to use the `-H` option with sudo, which will maintain proper permissions alongside proper separation between user and system files.

**Comment by CarolynWriter** [comment]
This is my TOP Priority right now to fix.

**Comment by CarolynWriter** [comment]
Rich, I was re-reading your comments . . . are you saying that the sudo advice for MacOS does not should be different from the sudo advice for Linux? So the -H option avoids the problems and issues you have with sudo and MacOS? I am the new doc writer at npm--I will be meeting with one of our engineers this week to really get this straight. Any other comments welcome. Thanks!

**Comment by richRemer** [comment]
No, the current sudo advice is bad advice for Linux and OS X. In either
case, the command will create root owned files in the user's home directory
and this will lead to subsequent permission issues that need to be fixed.
The updated advice resolves this issue for both Linux and OS X.

The reason I bring up Linux is that it is very common to run Linux servers
as shared machines between multiple users, but this is not as common on OS
X. The current advice does not address shared systems on either platform,
though.

**Comment by CarolynWriter** [comment]
Most of fhe doc has been updated with sudo removed.
